MekHQ/src/mekhq/campaign/force/ForceStub.java

```diff
@@ -1,7 +1,7 @@
 /*
  * Force.java
  * 
- * Copyright (c) 2011 Jay Lawson <jaylawson39 at yahoo.com>. All rights reserved.
+ * Copyright (c) 2018 the MegaMek Team.
```

Comment:
So many lines have changed that I guess Jay himself wouldn't recognise this file..